src/main/java/com/example/onlinebookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -1,22 +1,27 @@
 package com.example.onlinebookstore.dto.book;
 
 import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotEmpty;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
+import java.util.Set;
 import lombok.Data;
 import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
     @NotNull
+    @NotBlank
```

Comment:
![image](https://github.com/Zel1oy/online-book-store/assets/69054906/97703837-4b41-4f64-a7b8-2357a52f282b)
Some of these annotation are redundant